### PR TITLE
Allow parsing of ev inputs

### LINKED ISSUE #112: Allow parsing evs from PSDM `EvInput`

### LINKED ISSUE #113: Keep evs in sets instead of sorted sets
This was once to ensure replicable results in combination with seeded sampling. Since we now handle evs parallely this now does not work anymore

### PR DESCRIPTION
**Pull request by t-ober**
Resolve #113 first

Resolves #112

### CODE REVIEW

File: src/main/scala/edu/ie3/mobsim/MobilitySimulator.scala [hunk 1]
```diff
@@ -696,28 +696,50 @@ object MobilitySimulator
...
-      evModelPdf,
-      tripProbabilities.firstDepartureOfDay
+      )
+      .pdf
+
+    val workPoisWithSizes = poisWithSizes.getOrElse(
+      CategoricalLocationDictionary.WORK,
+      throw InitializationException(
+        "Unable to obtain the probability density function for work POI."
+      )
```
Comment (danielfeismann) [inline src/main/scala/edu/ie3/mobsim/MobilitySimulator.scala:712]: Shouldn't this be handled the same way as ´homePOIsWithSizes´ => with .pdf ?

**Review by danielfeismann — CHANGES_REQUESTED** [review]
To me this looks fine apart from one minor question and the missing changelog.

File: src/main/scala/edu/ie3/mobsim/MobilitySimulator.scala (hunk 1 above)
Comment (t-ober) [reply src/main/scala/edu/ie3/mobsim/MobilitySimulator.scala:712]: We split the home pois later on with regards of whether they can charge at home or not. Therefore we extract it from the original `ProbabilityDensityFunction` and build it for both possibilities again. With the work pdf we don't do such a thing.

**Review by danielfeismann — DISMISSED** [review]
Looks good, thanks a lot!

File: build.gradle [hunk 2]
```diff
@@ -38,7 +38,8 @@ repositories {
 }
 
 dependencies {
-  implementation('com.github.ie3-institute:PowerSystemUtils:2.0-SNAPSHOT') {
+  implementation('com.github.ie3-institute:PowerSystemUtils:2.0') {
+    exclude group: 'org.apache.logging.log4j'
```
Comment (sebastian-peter) [inline build.gradle:42]: Not the end of the world, but duplicate line

**Review by danielfeismann — APPROVED** [review]
LGTM

File: build.gradle [hunk 3]
```diff
@@ -41,8 +41,8 @@ dependencies {
   implementation('com.github.ie3-institute:PowerSystemUtils:2.0') {
     exclude group: 'org.apache.logging.log4j'
     exclude group: 'org.slf4j'
-    /* Exclude our own nested dependencies */
     exclude group: 'com.github.ie3-institute'
+    exclude group: 'com.github.johanneshiry', module: 'OSMonaut'
```
Comment (sebastian-peter) [inline build.gradle:45]: One more thing... We don't use OSMonaut anymore, if I remember correctly

**Review by danielfeismann — APPROVED** [review]

File: build.gradle (hunk 3 above)
Comment (danielfeismann) [reply build.gradle]: Yes, also regonized that. Should be removed as well

**Pull request merged**